server/controllers/person/viewPersonHome.ts

```diff
@@ -46,6 +46,21 @@ export default async (req: Request, res: Response) => {
     }
     // Nothing to do.
```

Comment:
minor and not changed in this PR - but this 'Nothing to do' comment in the code seems a bit odd